terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -130,6 +129,28 @@ data "aws_iam_policy_document" "member-access" {
     ]
     resources = ["*"]
   }
+
+  statement {
+    effect = "Deny"
+    actions = [
+      "iam:AttachRolePolicy",
+      "iam:DeleteRole",
+      "iam:DeleteRolePermissionsBoundary",
+      "iam:DeleteRolePolicy",
+      "iam:DetachRolePolicy",
+      "iam:PutRolePermissionsBoundary",
+      "iam:PutRolePolicy",
+      "iam:UpdateAssumeRolePolicy",
+      "iam:UpdateRole",
+      "iam:UpdateRoleDescription"
+    ]
+    condition {
```

Comment:
I don't think this condition is needed is it?

Reply:
Pulled the condition as discussed